src/main/io/osd.c

```diff
@@ -510,6 +514,19 @@ static bool osdDrawSingleElement(uint8_t item)
             break;
         }
 
+#ifdef USE_RX_LINK_QUALITY_INFO
+    case OSD_LINK_QUALITY:
+        {
+            // change range to 0-9 (two sig. fig. adds little extra value, also reduces screen estate)
+            uint8_t osdLinkQuality = rxGetLinkQuality() * 10 / LINK_QUALITY_MAX_VALUE;
+            if (osdLinkQuality >= 10)
```

Comment:
Sorry, didn't notice this before, but can you please put a block after the `if`? Using blocks even for single statements eliminates the risk of forgetting to add a block when more statements inside the condition are needed at a later stage (at that time the condition might be complex and hard to read).